README.md

```diff
@@ -92,17 +92,26 @@ mv /path/to/directory/role.prom.$$ /path/to/directory/role.prom
 
 
 ## Using Docker
-
-You can deploy this exporter using the [prom/node-exporter](https://registry.hub.docker.com/u/prom/node-exporter/) Docker image.
-
-For example:
+It's not recommended to deploy the node-exporter as Docker container because it
+requires access to the host system. If you need to run it on Docker, you can
```

Comment:
@SuperQ I went for a more technical description. Why do you think 'designed to monitor' is better than 'requires access to'? I imaging if we use the former, people might say: "well, designed for - but I still want to run it in Docker and Prometheus peepz don't allow it"